admin-dev/themes/new-theme/js/components/grid/extension/bulk-action-checkbox-extension.ts

```diff
@@ -38,10 +38,26 @@ export default class BulkActionCheckboxExtension {
    * @param {Grid} grid
    */
   extend(grid: Grid): void {
+    this.handleBulkActionCheckboxStatus(grid);
     this.handleBulkActionCheckboxSelect(grid);
     this.handleBulkActionSelectAllCheckbox(grid);
   }
 
+  /**
+   * Disable/Enable "Select all" button in the grid
+   *
+   * @param {Grid} grid
+   *
+   * @private
+   */
+  private handleBulkActionCheckboxStatus(grid: Grid) {
```

Comment:
Are you sure about this? From what I understand it disbaled the Select all checkbox until at leat one checbox is selected. But from a UX perspective it means you must first select a checkbox, then select the All checkbox, so it's one more click that may be unnecessary no?
We should be able to click on it right away, unless I'm missing something

Reply:
@Progi1984 @jolelievre 

Yes, bulk action checkbox should be enabled all the time, like it works on all other pages and on the legacy.

However, order statuses page is a bit special, because we don't have the checkbox for undeletable statuses. On legacy page, when there is no checkboxes (only undeletable statuses), no bulk actions button is shown.

![Snímek obrazovky 2023-07-28 101341](https://github.com/PrestaShop/PrestaShop/assets/6097524/ecde439a-f0ae-4db0-982a-0e5666d28b4a)

![Snímek obrazovky 2023-07-28 101602](https://github.com/PrestaShop/PrestaShop/assets/6097524/588781f5-2a3c-4210-9f3f-f78cb635fc4c)